### PR TITLE
Added wallet installation notes

### PR DESCRIPTION
**Pull request by dgarcia360**
Related issue #305

### CODE REVIEW

**Comment by evias** [comment]
Do we maybe want to point at `executables` being under `Assets` dropdown on releases on github?

**Comment by dgarcia360** [comment]
@evias Good point, added a note.

File: source/guides/network/creating-a-private-test-net.rst [hunk 1]
```diff
@@ -171,6 +147,7 @@ On the other hand, you can share securely the public and address of your account
 
 Now that you have your node running an account with some |privatenetworkcurrency|, we recommend you to surf the |sitename|. In this portal, you can find everything you need to know about |codename|'s features and :ref:`self-paced guides <blog-categories>` on how to use the **software development kits**.
```
Comment (matthewkim93) [inline source/guides/network/creating-a-private-test-net.rst:148]: Now that you have your node running an account with some |privatenetworkcurrency|, we recommend you to surf the |sitename|. In this portal, you can find detailed information about |codename|'s features and :ref:`self-paced guides <blog-categories>` on how to use the **software development kits**.

File: source/guides/transfer/sending-an-encrypted-message.rst [hunk 2]
```diff
@@ -34,7 +34,7 @@ Prerequisites
 Setting up the use case
 ***********************
 
-Create an account for Alice, and another for the certificate with the :doc:`CLI tool <../../cli>`.
+Create an account for Alice, and another for the certificate with the :ref:`CLI tool <wallet-cli>`.
```
Comment (matthewkim93) [inline source/guides/transfer/sending-an-encrypted-message.rst:37]: Create 2 accounts: one for Alice and another for the certificate with the :ref:`CLI tool <wallet-cli>`.

**Pull request merged**